Simplenote/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'io.sentry.android.gradle'
 
 android {
     buildToolsVersion '28.0.3'
-    compileSdkVersion 28
+    compileSdkVersion 29
```

Comment:
Since the version has been incremented to 29, shouldn't the `buildToolsVersion` be updated to `29.0.2`?

Reply:
Yep!  We can do that.  I updated the build tools version in a54ba7c2.